evalml/automl/automl_search.py

```diff
@@ -612,13 +613,21 @@ def _compute_cv_scores(self, pipeline, X, y):
             logger.debug(f"\t\tTraining and scoring on fold {i}")
             X_train, X_test = X.iloc[train], X.iloc[test]
             y_train, y_test = y.iloc[train], y.iloc[test]
+
             if self.problem_type in [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]:
+                try:
+                    rus = RandomUnderSampler(sampling_strategy='not minority', random_state=self.random_state, replacement=False)
+                    X_train, y_train = rus.fit_resample(X_train, y_train)
+                except ValueError:
+                    logger.warning(f"\t\t\tFold {i}: could not downsample to balance classification")
```

Comment:
Under what circumstances would this error be raised?

Reply:
I ran into it when there wasn't enough samples for each class. Maybe I can put a datacheck before this but I only went for a warning for now.

Reply:
@bchen1116 @dsherry @rpeck 

It seems imblearn does give an error if there are not enough samples. I think there are parameters to manage the number of samples as well.